pkg/controller/registry/reconciler/grpc.go

```diff
@@ -213,20 +211,10 @@ func (c *GrpcRegistryReconciler) currentPodsWithCorrectImageAndSpec(logger *logr
 		return nil, err
 	}
 	for _, p := range pods {
-		images, hash := correctImages(source, p), podHashMatch(p, newPod)
-		logger = logger.WithFields(logrus.Fields{
-			"current-pod.namespace": p.Namespace, "current-pod.name": p.Name,
-			"correctImages": images, "correctHash": hash,
-		})
-		logger.Info("evaluating current pod")
-		if !hash {
-			logger.Infof("pod spec diff: %s", cmp.Diff(p.Spec, newPod.Spec))
```

Comment:
This log seems like it could be useful in understanding why catalog pods are being recycled unexpectedly.